api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -22,7 +22,6 @@ import "google/protobuf/wrappers.proto";
 
 import "udpa/core/v1/resource_locator.proto";
 
-import "envoy/annotations/deprecation.proto";
```

Comment:
removing this triggers an error

```
ERROR: /build/tmp/_bazel_bazel/b570b5ccd0454dc9af9f65ab1833764d/external/envoy_api/envoy/config/bootstrap/v3/BUILD:7:18: Generating Descriptor Set proto_library @envoy_api//envoy/config/bootstrap/v3:pkg failed (Exit 1) protoc failed: error executing command bazel-out/host/bin/external/com_google_protobuf/protoc '--proto_path=bazel-out/k8-fastbuild/bin/external/com_google_protobuf/_virtual_imports/any_proto' ... (remaining 178 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
envoy/config/bootstrap/v3/bootstrap.proto:276:27: Option "(envoy.annotations.disallowed_by_default)" unknown. Ensure that your proto definition file imports the proto which defines the option.
```

not sure if this is my bad...

Reply:
Yeah, you need that one. Was it generating a warning before?

Reply:
i think so - but i need to check. it seems like there are others that are also required - which is making it quite slow to go through.

Ill update as i have time...